### PR TITLE
Changes to the interface and adds responsive features

### PR DESCRIPTION
**Pull request by EmmanuelBoM**
This PR makes changes to the interface of landing page and offer gallery, as well as adds responsive features to work in diverse screen sizes.
image
image

### CODE REVIEW

File: team_project/src/main/webapp/offerGallery.css [hunk 1]
```diff
@@ -1,18 +1,31 @@
...
+}
+
+.title{
+    font-family: 'Montserrat';
 }
 
+.title span {
+    font-weight: bold;
+    background: linear-gradient(to right, #BF80E9 0%, #E48DDE 53%);
+    background-clip: text;
```
Comment (supercontact) [inline team_project/src/main/webapp/offerGallery.css:26]: This does not work for my Chrome browser, but -webkit-background-clip works. Do you need to add that or does bootstrap do some polyfill when deployed?

File: team_project/src/main/webapp/styles.css [hunk 2]
```diff
@@ -1,48 +1,205 @@
...
+.description-text{
+    color: #4d4d4d;
+    margin-bottom: 30px;
+    text-align: justify;
+}
+
+span {
+    font-weight: bold;
+    background: linear-gradient(to right, #FC77AE 0%, #ff9085 100%);
+    background-clip: text;
```
Comment (supercontact) [inline team_project/src/main/webapp/styles.css:37]: Same comment^

**Review by supercontact — APPROVED** [review]
We have an awesome-looking web page now!😃

File: team_project/src/main/webapp/offerGallery.css (hunk 1 above)
Comment (EmmanuelBoM) [reply team_project/src/main/webapp/offerGallery.css:26]: Yeah, looks like -webkit- fixes the issue. Just added a new commit that does that.

**Pull request merged**